### PR TITLE
adding menu mobile

### PR DESCRIPTION
**Pull request by boo-shehab**
we just completed the mobile menu for my portfolio, project and we added the following:

- Added Js file
- HTML,CSS and Js Linters checked
- Added a descriptive Readme file
- Added menu for the mobile version

### CODE REVIEW

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @boo-shehab , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

1. 🛑 You have submitted a project, that doesn't have javascript linters installed, kindly add [Javascripts Linters](https://github.com/microverseinc/linters-config/tree/master/javascript) with your HTML and CSS linters 
2. 🛑 as mentioned in the requirements this is a pair programming project and all of your group members need to commit in your project 

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: css/style.css [hunk 1]
```diff
@@ -150,15 +164,17 @@ header #burger-button:hover + nav {
   top: 0;
   background-color: #6070ff;
   mix-blend-mode: multiply;
+  z-index: 3;
 }
```
Comment (AmaduKamara) [inline css/style.css:168]: ## Recommendation Change
This change will help improve the way your app looks. It's not a significant change to ask for a fix and resubmit for review but I trust you can fix it.

- [ ] To complete the blurred background for the mobile menu background, kindly add this opacity style below these styles. `opacity: .8;`. What do you think? 👍 

Adding that style to the `header .burger-button.active + nav{}` will give you a look similar to the original template as required.

image

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢🟢🟢 

Hi @boo-shehab,

Your project is complete!  

## Highlights:
✅ You used Gitflow correctly
✅ Linter checks passed
✅ Good descriptive PR 👍
✅ Mobile menu works fine as expected 👍

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: css/style.css (hunk 1 above)
Comment (boo-shehab) [reply css/style.css:168]: I think you are a genius I did that thanks for the suggestion

**Comment by AmaduKamara** [comment]
Good job 👍
Keep doing great bro

**Pull request merged**